scripts/install2

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env bash
+# s_h_e_l_lcheck disable=SC1091
+# true
```

Comment:
?

Reply:
it was a shellcheck workarond 

to disable a specific shellcheck check for the whole file you do
```
# shellcheck disable=SC1091
true
```